RetailCoder.VBE/Inspections/InspectionsUI.resx

```diff
@@ -555,4 +555,13 @@
   <data name="VariableTypeNotDeclaredInspectionResultFormat" xml:space="preserve">
     <value>{0} '{1}' is implicitly 'Variant'</value>
   </data>
-</root>
+  <data name="MalformedAnnotationInspectionMeta" xml:space="preserve">
+    <value>An annotation comment is malformed.</value>
+  </data>
+  <data name="MalformedAnnotationInspectionName" xml:space="preserve">
+    <value>Malformed annotation.</value>
```

Comment:
Well, it is recognized as an `'@Ignore` or `'@Folder` annotation.  This does *not* warn for things like `'@jreifkl`, which would truly be an "unrecognized" annotation.  I chose malformed because these two annotations require arguments, and the arguments were not supplied.